environment.yml

```diff
@@ -0,0 +1,34 @@
+# To use:
+#   $ conda env create -f environment.yml  # `mamba` works too for this command
+#   $ conda activate numpy-dev
+name: numpy-dev
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - cython
```

Comment:
```suggestion
  - cython=0.29.22
```

Reply:
No, I only want to pin things that are very likely to cause problems. This is unnecessary, and also wrong in the corresponding requirements.txt file.

Reply:
A more explicit name would help, maybe `conda-environment.yml`?

Reply:
`environment.yml` is the standard name for this file in other projects and in educational material (example: [SW Carpentry](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/04-sharing-environments/index.html)). It's also what IDEs and tools like GitHub will look for if they gain conda support. So renaming is not a good idea.

Analogy: you wouldn't call something `pypi_requirements.txt` either.

Reply:
Pandas equivalent: https://github.com/pandas-dev/pandas/blob/master/environment.yml